atomicapp/providers/openshift.py

```diff
@@ -593,10 +610,11 @@ def _get_url(self, namespace, kind, name=None, params=None):
         if name:
             url = urljoin(url, name)
 
-        if params:
-            params["access_token"] = self.access_token
-        else:
-            params = {"access_token": self.access_token}
+        if not params:
+            params = {}
```

Comment:
just set `params={}` in the function declaration. That way `{}` is the default value given if params isn't provided right? I guess someone could provide params=None though :(